building-pyramids/java/printingloops2.java

```diff
@@ -0,0 +1,16 @@
+public class printingloops2{
+    public static void main(String[] args){
+        int count = 1;
+        String dot = "*";
+        String space = "                                        ";
+        
+        while (count < 40){
+           
+            System.out.print(space);
+            space = space.concat("\b");
```

Comment:
You could just remove characters from the end of the string like this:
space = space.substring(0, space.length() - 1);